tools/tools.go

```diff
@@ -0,0 +1,26 @@
+// +build tools
+
+// To install the following tools at the version used by this repo run:
+// $ make tools
+// or
+// $ go generate -tags tools tools/tools.go
+
+package tools
+
+//go:generate go install github.com/hashicorp/go-bindata
+import _ "github.com/hashicorp/go-bindata"
+
+//go:generate go install github.com/golang/protobuf/proto
+import _ "github.com/golang/protobuf/proto"
+
+//go:generate go install github.com/golang/protobuf/protoc-gen-go
+import _ "github.com/golang/protobuf/protoc-gen-go"
+
+//go:generate go install github.com/grpc-ecosystem/grpc-gateway
+import _ "github.com/grpc-ecosystem/grpc-gateway"
```

Comment:
Quick question on this...it seems like the `mitchellh/protoc-gen-go-json` package requires this? But I don't think I'm setting it up right because it is excluding the go files :sweat_smile: 

```
==> github.com/grpc-ecosystem/grpc-gateway
package github.com/grpc-ecosystem/grpc-gateway: build constraints exclude all Go files in /home/brian/go/src/github.com/grpc-ecosystem/grpc-gateway
==> github.com/mitchellh/protoc-gen-go-json
package github.com/grpc-ecosystem/grpc-gateway/codegenerator: cannot find package "github.com/grpc-ecosystem/grpc-gateway/codegenerator" in any of:
        /usr/lib/go-1.14/src/github.com/grpc-ecosystem/grpc-gateway/codegenerator (from $GOROOT)
        /home/brian/go/src/github.com/grpc-ecosystem/grpc-gateway/codegenerator (from $GOPATH)
package github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor: cannot find package "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor" in any of:
        /usr/lib/go-1.14/src/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor (from $GOROOT)
        /home/brian/go/src/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor (from $GOPATH)
```

Reply:
It does require that... it should just do it automatically without you having to have this import.

Reply:
Without including it, I end up with the same error saying that it's missing :thinking:

Reply:
Oddly enough, enabling `GO11MODULE` with go generate doesn't run into this :thinking: Maybe we go with this enabled instead for `make tools`? (related to the other comment on PR)